config.toml.example

```diff
@@ -391,8 +391,7 @@
 # desired in distributions, for example.
 #rpath = true
 
-# Emits extraneous output from tests to ensure that failures of the test
-# harness are debuggable just from logfiles.
+# Emits extra output from tests so test failures are debuggable just from logfiles.
```

Comment:
Quoting myself from https://github.com/rust-lang/rustc-dev-guide/pull/795#discussion_r479763164:
> This change doesn't seem right, specifically "test harness" refers to *the code running the tests*, rather than the tests themselves. And it is extraneous for the common case of e.g. `libtest` not crashing.

cc @Mark-Simulacrum

Reply:
verbose tests indeed changes nothing about tests, only the harnesses (compiletest and libtest) which invoke them.